README.md

```diff
@@ -122,6 +122,13 @@ Whenever you want to contribute to any project it is a good practice to make a s
 
 This bot is reserved for functionalities offered for community classroom community.
 
-### Maintainer :
+## Contributors :
+
+Every contributor's efforts and time are deeply appreciated :smile:
 
+<a href = "https://github.com/kaiwalyakoparkar/classroom-monitor-bot/graphs/contributors">
+  <img src = "https://contrib.rocks/image?repo=kaiwalyakoparkar/classroom-monitor-bot"/>
+</a>
+
+### Maintainer :

```

Comment:
Remove this maintainer section as you added the contributor's list already :smile:

Reply:
Done! @kaiwalyakoparkar